src/psij/executors/batch/test/qrun

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+import os
+import random
+import subprocess
+import sys
+import time
+
+from qlib import *
+
+job_id = sys.argv[1]
+script = sys.argv[2]
+
+
+logger.info('Running job %s', job_id)
+wait_time = random.randint(1, 10)
+
+logger.info('Queue time for %s is %s', job_id, wait_time)
+#time.sleep(wait_time)
+time.sleep(0.11)
```

Comment:
Assuming this was a local change for debugging that slipped in:

```suggestion
time.sleep(wait_time)
```

Reply:
ooops!

Reply:
Reverted